velox/type/TimestampConversion.cpp

```diff
@@ -749,4 +750,13 @@ Timestamp fromTimestampString(const char* str, size_t len) {
   return timestamp;
 }
 
+std::tuple<const StringView&, std::string_view>
+splitTimestampWithTimezoneString(const std::string_view& tsTzStr) {
+  auto pos = tsTzStr.find(" ");
+  pos = tsTzStr.find(" ", pos + 1);
+  auto tsStr = tsTzStr.substr(0, pos);
+  auto tzStr = tsTzStr.substr(pos + 1);
+  return std::make_tuple(StringView(tsStr), (std::string_view)(tzStr));
```

Comment:
`tsStr` and `tzStr` are local to the function. We cannot return StringView of them.

Reply:
@majetideepak Thanks for catching that - I changed the `const StringView&` reference in the tuple to return a value instead, to avoid issues with that reference potentially becoming inaccessible. 

Because the input `tsTzStr` is coming from the caller, and the returned values are nothing but references to that string (via `substr()`), I think there should be no issues with access outside the function now.